app/services/database.js

```diff
@@ -162,6 +165,7 @@ export default Ember.Service.extend(PouchAdapterUtils, {
   _createRemoteDB(remoteUrl, pouchOptions) {
     return new Ember.RSVP.Promise(function(resolve, reject) {
       let remoteDB = new PouchDB(remoteUrl, pouchOptions);
+      window.remoteDB = remoteDB;
```

Comment:
Using a global is unneeded.  The application adapter has access to the database service, so it can use that to get the remoteDb via:
```js
this.get('database').get('mainDB')
```